xcm/src/v1/multilocation.rs

```diff
@@ -898,7 +895,7 @@ mod tests {
 		);
 		assert_eq!(
 			v0::MultiLocation::X2(v0::Junction::Parachain(88), v0::Junction::Parent).try_into(),
-			Err::<MultiLocation, ()>(()),
+			Ok(MultiLocation::here()),
```

Comment:
why is this not an error anymore? because of `v0.canonicalize();`?

Reply:
Correct, what results after canonicalization is exactly `Here`.